### DISCUSSION TITLE
Funscript - how to work ?

### DISCUSSION THREAD

**Discussion by davnozdu** [stashapp/stash#5226] [Q&A] [answered]
Hello. 
I buy Handy device but I not understand how to work Funscript ?
In stash I activate function for generate funscript and enter API key for Handy.
Stash can generate Funscript for video or I can use only Funscript file from internet ?
If need make manually, anybody know python script or Docker image for this or maybe library Funscript from community with stash integration ?

**Comment by DogmaDragon** [comment, marked as answer]
Stash only syncs Handy with existing funscript files, doesn't create them.

> Funscript files must be in the same directory as the matching video file and must have the same base name. For example, a funscript file for video.mp4 must be named video.funscript. A scan must be run to update scenes with matching funscript files.

https://docs.stashapp.cc/in-app-manual/interactive/
